---
layout: posts
title: Educational Resources for for Amplicon Metagenomics
category: Curated External Sources
excerpt: A curated list of articles, tutorials and workshops for microbiome data processing and analysis, and more.
tags:
    - educational
    - amplicon-metagenomics
    - resource
    - curated-external
    - tutorial
    - workshop
    - article
---

Articles
--------
------------------------------------------------------------------------
1.  **mothur and QIIME**
    -   **Description:** Very nice article that describes how mothur and
        QIIME are different and/or similar.
    -   **Link:** <http://blog.mothur.org/2016/01/12/mothur-and-qiime>

Workshops
---------
------------------------------------------------------------------------
1.  **CoDaSeq workshop**
    -   **Description:** The intention of this workshop is to give
        participants a background in Compositional Data Analysis (CoDa)
        for transcription studies. At the end of the workshop you will
        have been introduced to CoDa, why you’d use CoDa, and gain
        hands-on experience using CoDa analysis in R.
    -   **Link:**
        <https://github.com/ggloor/CoDa_microbiome_tutorial/wiki>
2.  **Microbiota Analysis in R \[workshop/tutorial\]**
    -   **Description:** The goal of this tutorial is to demonstrate
        basic analyses of microbiota data to determine if and how
        communities differ by variables of interest. In general, this
        pipeline can be used for any microbiota data set that has been
        clustered into operational taxonomic units (OTUs).
    -   **Link:** <https://rpubs.com/dillmcfarlan/R_microbiotaSOP>

Tutorials
---------
------------------------------------------------------------------------

1.  **Microbiota Procesing in mothur: standard operating procedure
    (SOP)**
    -   **Description:** The goal of this tutorial is to demonstrate
        standard operating procedures (SOP) for processing amplicon data
        that are generated using Illumina’s MiSeq platform with paired
        end reads.
    -   **Link:** <https://rpubs.com/dillmcfarlan/mothurSOP>
2.  **16S Metagenomic Analysis Tutorial**
    -   **Description:** In this tutorial we will explore the primary
        characterization of a specific human gut microbiome dataset,
        namely the enterotype dataset from...
    -   **Link:**
        [http://web.evolbio.mpg.de/~wang/Site/R\_tutorial\_files/16S Metagenomic Analysis Tutorial.pdf](http://web.evolbio.mpg.de/~wang/Site/R_tutorial_files/16S Metagenomic Analysis Tutorial.pdf)
3.  **An Introduction to Applied Bioinformatics\[OnlineBook\]**
    -   **Description:** An Introduction to Applied Bioinformatics (or
        IAB) is a free, open source interactive text that introduces
        readers to core concepts of bioinformatics in the context of
        their implementation and application.
    -   **Link:** <http://readiab.org>
    ***Focus mainly on following chapters***
    1.  **Phylogenetic reconstruction**
        -   **Description:** Explore the goals, approaches, and
            challenges for creating phylogenetic trees, or phylogenies.
        -   **Link:** <http://readiab.org/book/0.1.3/2/4>
    2.  **Studying Microbial Diversity**
        -   **Description:** Explore terminology, methods and approaches
            for performing microbial diversity analysis.
        -   **Link:** <http://readiab.org/book/0.1.3/3/1>
4.  **Microbiome data analysis**
    -   **Description:** The purpose of this tutorial is to describe the
        steps required to perform microbiome data analysis, explain how
        to build your own data flow, and finally, discuss the results
        obtained in such analysis.
    -   **Link:**
        <https://genestack-user-tutorials.readthedocs.io/tutorials/Microbiome_analysis>
5.  **QIIME2 Tutorials**

    1.  **QIIME2 Glossary**
        -   **Description:** It is important to first know QIIME2
            terminology before starting to learn it.
        -   **Link:** <https://docs.qiime2.org/2019.1/glossary>
    2.  **Overview of QIIME2 Plugin Workflows**
        -   **Description:** After terminology this tutorial is your
            first step to learn QIIME2.
        -   **Link:**
            <https://docs.qiime2.org/2019.1/tutorials/overview/>
    3.  **Microbiome Analysis with QIIME2: A Hands-On
        Tutorial\[Presentation\]**
        -   **Description:** Nice tutorial for introduction to QIIME2
        -   **Link:**
            <http://compbio.ucsd.edu/wp-content/uploads/2018/07/20180621_oslo_university_microbiome_analysis_with_qiime2_tutorial.pdf>
    4.  **“Moving Pictures” tutorial \[VeryNiceTutorial!\]**
        -   **Description:** In this tutorial you’ll use QIIME 2 to
            perform an analysis of human microbiome samples from two
            individuals at four body sites at five timepoints, the first
            of which immediately followed antibiotic usage.
        -   **Link:**
            <https://docs.qiime2.org/2019.1/tutorials/moving-pictures>
    5.  **QIIME2 workflow**
        -   **Description:** Using the Quantitative Insights Into
            Microbial Ecology (QIIME2) software pipeline for analysis of
            marker gene-based microbiome sequencing data
        -   **Link:** <https://chmi-sops.github.io/mydoc_qiime2.html>
6.  **Amplicon Metagenomics**
    -   **Description:** This tutorial describes a strategy for
        assembling, filtering and analysing an NGS metagenomic data set
        in Geneious Prime.
    -   **Link:**
        <https://www.geneious.com/tutorials/metagenomic-analysis>
7.  **Taxonomy from species name in R**
    -   **Description:** This tutorial describes one of many approaches
        to generate phylogenetic tree from specie names.
    -   **Link:**
        <https://taylorreiter.github.io/2017-07-28-Taxonomy-from-Species-Name-in-R>
