---
layout: publication
title: Software and Tools for Amplicon Metagenomics
tags: microbiome amplicon-metagenomics tools packages software curated-external phylogenetic
category: Curated External Sources
date: 2019-09-03
output: true
description: A curated list of list of software, web-apps and R/Python packages, which can be used in microbiome research, and more.
---
WebApps/WebTools
----------------
------------------------------------------------------------------------

1.  **Phylo.io**
    -   **Description:** Nice and clean web application for visualizing
        phylogenetic trees.
    -   **Link:** <http://phylo.io/>
2.  **MicrobiomeAnalyst**
    -   **Description:** MicrobiomeAnalyst is a comprehensive
        statistical, visual and meta-analysis of microbiome data.
    -   **Link:** <https://www.microbiomeanalyst.ca>

Software and Tools
------------------
------------------------------------------------------------------------

1.  **QIIME2**
    -   **Description:** QIIME2 is a next-generation microbiome
        bioinformatics platform that is extensible, free, open source,
        and community developed.
    -   **Link:** <https://docs.qiime2.org/2019.7/install>
2.  **mothur**
    -   **Description:** mothur aims to be a comprehensive software
        package that allows users to use a single piece of software to
        analyze community sequence data.
    -   **Link:**
        <https://github.com/mothur/mothur/releases/tag/v.1.43.0>
3.  **KRAKEN**
    -   **Description:** Kraken is an ultrafast and highly accurate
        program for assigning taxonomic labels to metagenomic DNA
        sequences.
    -   **Link:** <http://ccb.jhu.edu/software/kraken>
4.  **USEARCH**
    -   **Description:** USEARCH is a unique sequence analysis tool with
        thousands of users world-wide. USEARCH offers search and
        clustering algorithms that are often orders of magnitude faster
        than BLAST.
    -   **Link:** <https://www.drive5.com/usearch>
5.  **VSEARCH**
    -   **Description:** VSEARCH is an open source and free of charge
        multithreaded 64-bit tool for processing and preparing
        metagenomics, genomics and population genomics nucleotide
        sequence data. It is designed as an alternative to the widely
        used USEARCH tool.
    -   **Link:** <https://github.com/torognes/vsearch>
6.  **MG-RAST**
    -   **Description:** MG-RAST is an open source, open submission web
        application server that suggests automatic phylogenetic and
        functional analysis of metagenomes.
    -   **Link:** <https://www.mg-rast.org>
7.  **PICRUSt**
    -   **Description:** PICRUSt (pronounced “pie crust”) is a
        bioinformatics software package designed to predict metagenome
        functional content from marker gene (e.g., 16S rRNA) surveys and
        full genomes.
    -   **Link:** <http://picrust.github.io/picrust>
8.  **pplacer**
    -   **Description:** Pplacer places query sequences on a fixed
        reference phylogenetic tree to maximize phylogenetic likelihood
        or posterior probability according to a reference alignment.
    -   **Link:** <https://matsen.fhcrc.org/pplacer>
9.  **The Exelixis Lab \[Curated Collection of Software/Tools\]**
    -   **Description:** Include software/tools for:
        -   *Phylogenetic Inference*
        -   *Phylogenetic Post-Analysis*
        -   *Next Generation Sequencing & Sequence Analysis*
        -   *Population genetics*
        -   *Web services*
        -   *etc.*
    -   **Link:** <https://cme.h-its.org/exelixis/software.html>

R packages
----------
------------------------------------------------------------------------

1.  **microbiome**
    -   **Description:** Tools for the exploration and analysis of
        microbiome profiling data sets, in particular large-scale
        population studies and 16S taxonomic profiling.
    -   **Tutorial:** <https://microbiome.github.io/tutorials>
    -   **Link:** <https://github.com/microbiome/microbiome>
2.  **microbiomeSeq**
    -   **Description:** An R package for microbial community analysis
        in an environmental context
    -   **Tutorial:**
        <http://userweb.eng.gla.ac.uk/umer.ijaz/projects/microbiomeSeq_Tutorial.html>
    -   **Link:** <https://github.com/umerijaz/microbiomeSeq>
3.  **matR - Metagenomics Analysis Tools**
    -   **Description:** An analysis platform for metagenomics combining
        specialized tools and workflows, easy handling of the BIOM
        format, and transparent access to MG-RAST resources. Integrates
        easily with other R packages and non-R software.
    -   **Tutorial:**
        <https://www.mcs.anl.gov/~braithwaite/matR/docs-and-pubs/matR-user-manual.pdf>
    -   **Link:** <https://rdrr.io/cran/matR>
4.  **MonoPhy**
    -   **Description:** The R package MonoPhy allows assessment and
        exploration of monophyly of taxa in a phylogeny.
    -   **Tutorial:**
        <https://cran.r-project.org/web/packages/MonoPhy/vignettes/MonoPhyVignette.html>
    -   **Link:**
        <https://cran.r-project.org/web/packages/MonoPhy/index.html>
